docs/source/getting-started.md

```diff
@@ -310,7 +327,7 @@ Note on **chain certificates**: If you are using a chain certificate, see also
 Note: In certain cases, e.g. **behind SSL termination in nginx**, allowing no SSL
 running on the hub may be desired.
 
-### Cookie secret
+### Cookie secret TODO Does this change at all with OAuth2?
```

Comment:
For the most part, no. However, the sentence below:

> If this value changes for the Hub, all single-user servers must also be restarted.

Should be removed because it is no longer true (I think maybe it never was).